src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -17,16 +17,49 @@ class IntroductionPage extends React.Component {
   render() {
     return (
       <div className="schemaform-intro">
-        <FormTitle title="Apply for pension with the 21-527EZ"/>
-        <p>
-          Fill out this application with the most accurate information you have. The more accurate it is, the more likely you are to get a rapid response.
-        </p>
-        <p>
-          VA uses the information you submit to determine your eligibility and to provide you with the best service.
-        </p>
-        <p>
-          Federal law provides criminal penalties, including a fine and/or imprisonment for up to 5 years, for concealing a material fact or making a materially false statement. (See <a href="https://www.justice.gov/usam/criminal-resource-manual-903-false-statements-concealment-18-usc-1001" target="_blank">18 U.S.C. 1001</a>)
-        </p>
+        <FormTitle title="Apply for pension benefits"/>
+        <p>This application is equivalent to Form 21-527EZ (Application for Pension).</p>
+        <SaveInProgressIntro
+            pageList={this.props.route.pageList}
+            resumeOnly
+            {...this.props.saveInProgressActions}
+            {...this.props.saveInProgress}/>
+        <div className="process schemaform-process schemaform-process-sip">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>What you need to fill out this application</h6></div>
+              <ul>
+                <li>Social Security number (required)</li>
+                <li>Military history (required)</li>
+                <li>Your financial information and the financial information of your dependents (required)</li>
+                <li>Work history</li>
+                <li>Bank account direct deposit information</li>
+                <li>Medical information</li>
+              </ul>
+              <p><a href="http://www.va.gov/ogc/apps/accreditation/index.asp">An accredited representative</a> with a Veterans Service Organization (VSO) can help you pick the right program.</p>
```

Comment:
It [looks](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3611#issuecomment-312110401) like this should be "fill out the claim".